charts/Chart.yaml

```diff
@@ -1,7 +1,17 @@
 apiVersion: v2
 name: opensearch-operator
-description: A Helm chart for Kubernetes
+description: A Helm chart for Kubernetes OpenSearch Operator
 
+home: https://github.com/Opster/opensearch-k8s-operator
+icon: https://opster.com/wp-content/uploads/2022/04/Opster_Logo_Rectangle_Dark.svg
+maintainers:
+  - email: idan@opster.com
+    name: Idan levi
+  - email: ori.g@opster.com
+    name: Ori Goldstein
+sources:
+  - https://github.com/Opster/opensearch-k8s-operator
+version: "1.1"
```

Comment:
This should correspond to the version of the operator

Reply:
changed